src/winzone/WindowsTimeZoneIDs.jl

```diff
@@ -34,30 +35,67 @@ function compile(xml_file::AbstractString, translation_file::AbstractString)
         end
     end
 
+    return translation
+end
+
+function compile(xml_file::AbstractString, translation_file::AbstractString)
     open(translation_file, "w") do fp
-        serialize(fp, translation)
+        serialize(fp, compile(xml_file))
```

Comment:
Serialization is used here to avoid having to re-parse the XML during the package loading. I don't need to strictly need to use serialization in this case.

I do use serialization for storing the TimeZone types. Can serialization change between Julia patch releases?